.circleci/config.yml

```
@@ -31,9 +31,27 @@ jobs:
           command: |
             python ./src/sync_dashboards/main.py sync --config lookml_dashboards.yaml
           no_output_timeout: 15m
+  validate-lookml:
+    docker:
+      - image: circleci/node
+    steps:
+      - checkout
+      - run:
+          name: Execute linting
+          command: |
+            npm set prefix ~/.npm
+            PATH="$HOME/.npm/bin:$PATH"
+            PATH="./node_modules/.bin:$PATH"
+            source ~/.bashrc
+            npm install -g @looker/look-at-me-sideways;
```

The next-generation `cimg/node` image appears to allow global npm package installs, so when using that image messing with the npm prefix and path shouldn't be necessary.

I'd also recommend pinning the `look-at-me-sideways` package to at least the major version to avoid unexpected behavior changes.

```suggestion
            npm install -g @looker/look-at-me-sideways@3
```